enhancements/update/change-management-and-maintenance-schedules.md

```diff
@@ -0,0 +1,1220 @@
+operational aspects are expected to be introduced.
+
+## Support Procedures
+
+Change management problems created by faulty implementations will need to be resolved by 
+analyzing operator logs. The operator responsible for a given resource will vary. Existing
+support tooling like must-gather should capture the information necessary to understand
+and fix issues. 
+
+Change management problems where user expectations are not being met are designed to
+be informed by the detailed `status` provided by the resources bearing the `changeManagement`
+stanza in their `spec`.
+
+## Alternatives
+
+### Implement maintenance schedules via an external control system (e.g. ACM)
+We do not have an offering in this space. ACM is presently devoted to cluster monitoring and does
+not participate in cluster lifecycle.
+
+### Do not separate control-plane and worker-node updates into separate phases
+As separating control-plane and worker-node updates into separate phases is an important motivation for this
+enhancement, we could abandon this strategic direction. Reasons motivating this separation are explained 
+in depth in the motivation section.
+
+### Separate control-plane and worker-node updates into separate phases, but do not implement the change control concept
+As explained in the motivation section, there is a concern that implementing this separation without
+maintenance schedules will double the perceived operational overhead of OpenShift updates. 
+
+This also creates work for our Service Delivery team without any platform support.
+
+### Separate control-plane and worker-node updates into separate phases, but implement a simpler MaintenanceSchedule strategy
+We could implement change control without `disabledUntil`, `pausedUntil`, `exclude`, and perhaps more. However,
+it is risky to impose a single opinionated workflow onto the wide variety of consumers of the platform. The workflows
+described in this enhancement are not intended to be exotic or contrived but situations in which flexibility
+in our configuration can achieve real world, reasonable goals. 
+
+`disabledUntil` is designed to support our Service Delivery team who, on occasion, will need
+to be able to bypass configured change controls. The feature is easy to use, does not require 
+deleting or restoring customer configuration (which may be error-prone), and can be safely 
+"forgotten" after being set to a date in the future.
+
+`pausedUntil`, among other interesting possibilities, offers a cluster lifecycle administrator the ability
+to stop a problematic update from unfolding further. You may have watched a 100 node
+cluster roll out a bad configuration change without knowing exactly how to stop the damage
+without causing further disruption. This is not a moment when you want to be figuring out how to format
+a date string, calculating timezones, or copying around cluster configuration so that you can restore
+it after you stop the bleeding.

```

Comment:
I've got nearly 1200 lines into this EP and only just worked out the difference between these. Perhaps I missed it earlier, but may want to make this clearer up top somewhere.

I also think reviewing the names might be worthwhile, naming is hard but I find the names aren't different enough right now to make it obvious what the use cases are

Reply:
arrg.. Sorry. Yep. `disabled` is a reference to whether change management is being enforced. `paused` is talking about reconciliation processes and named after the existing HyperShift API.

Reply:
Might just be worth adding a detailed description near the introduction of the two terms near the front of the document so other readers get that understanding earlier

Reply:
Added `Definitions and Reference` section: https://github.com/openshift/enhancements/pull/1571/commits/ec9eee7d34fea72352fee695704cc598a5e5e8c1#diff-1ff326e1e96737c91e17f11d7852c3a9e3510415ea04892601c31a867e6eb706R60 .